src/SimpleHypergraphs.jl

```diff
@@ -8,10 +8,12 @@ export set_vertex_meta!, get_vertex_meta
 export set_hyperedge_meta!, get_hyperedge_meta
 export BipartiteView, shortest_path
 export TwoSectionView
+export modularity, randompartitioning, findmodularity
```

Comment:
why `randompartitioning` is exported?

Reply:
we are planning more API with regard to modularity. This could be useful for demos, tutorials etc.

Reply:
maybe `randompartition` then?

Reply:
corrected